job-tutorial/README.md

```diff
@@ -146,6 +148,7 @@ Below we show a complete list of environment variables accessible in a Docker co
 | :--------------------------------- | :--------------------------------------- |
 | PAI_WORK_DIR                       | Working directory in Docker container    |
 | PAI_DEFAULT_FS_URI                 | Default file system uri in PAI           |
+| PAI_JOB_VC_NAME                    | `virtualCluster` in config file          |
```

Comment:
>virtualCluster [](start = 40, length = 14)

you mean the virtual cluster the job runs on?

Reply:
Yes. Here it means the virtual cluster job runs on